### PR TITLE
feat: added cards section to component library

### PR DESCRIPTION
**Pull request by abhijeetekad**
CPT2202030245-1875x877
Cards added:
1)Normal card.
2)vertical card.
3)Horizontal card.
4)text overlay card.
5)card with a badge.
6)dismiss card.
7) horizontal shopping card.
8)youtube thumbnail card.

### CODE REVIEW

File: components/cards.css [hunk 1]
```diff
@@ -0,0 +1,564 @@
+@import url("/documentation/doc.css");
+.cards {
+  display: flex;
+  flex-direction: column;
+}
+.cards-shape {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 2rem 0rem 2rem 0rem;
```
Comment (devananth) [inline components/cards.css]: For this you can use shorthand , `margin : top&botton left&right`. In this case ,
 it would like `margin : 2rem 0rem`;

**Review by ashishnnnnn — APPROVED** [review]
Code seems fines and preview looks good

File: components/cards.css [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+}
+.price span {
+  color: red;
+}
+.portrait-card-btn .btn1 {
+  font-size: 18px;
+  font-weight: 600;
+  padding: 12px;
+  margin-left: 10px;
+  background-color: #ff9900;
```
Comment (prathmeshjagtap) [inline components/cards.css:312]: background-color property is repeating at a lot of places try to declare that color in your main CSS file and use variable naming to access the color.

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

File: components/cards.css (hunk 2 above)
Comment (abhijeetekad) [reply components/cards.css:312]: @prathmeshjagtap made changes as you suggested. actually, I started using the same class for a similar component but I was making more mistakes so I stopped using the same class but will use it from now.

File: components/cards.css (hunk 1 above)
Comment (abhijeetekad) [reply components/cards.css]: hey, @devananth changes have been done according to shorthand property. will use everywhere. Thanks for the feedback.

**Pull request merged**